### ISSUE TITLE
Is there a way to destroy the object ?

### ISSUE THREAD

**Issue by ch0rn** [MakG10/jquery-meme-generator#7]
Sorry if this is written poorly.  I am still learning javascript/jquery.

I have a project I am working on where I have a canvas where users can make several different modifications to it.  One of which would be the meme generator.  Problem is I need the meme generator to be initialized, then once the image is saved, destroy the object.  It will then return the saved image back to the original canvas.  Is there a function where the meme generator can be un-initialized / destroyed ? or at least turned off, then back on later ?

Thanks for your help, this is a really nice piece of work here.

**Comment by MakG10** [comment]
There is no destroy method at the moment, but there should be, I will keep it in mind for the next release.

Right now you could simply remove the `.mg-wrapper` container, but first restore your `<img>` tag - either create new one with the same attributes or move the one in `.mg-wrapper`
```javascript
var container = $('..mg-wrapper');
container.find('mg-image > img').appendTo(container.parent());
container.remove();
```

Meme Generator doesn't modify the source `<img>` tag in any way. If you want the canvas with captions, you have to call `saveCanvas` method first.

**Comment by ch0rn** [comment]
I think I can make it work this way.  Thank you for the help.

**Comment by ch0rn** [comment]
I hate to bother you again.  I was able to get it mostly working with your suggestion.  I have run into only 1 slightly minor problem still.

User can enter meme generator, add a meme.  Click save, it would then restore them back to a different canvas and hides the meme generator.  Looks nice and seamless.  Problem now is when the user goes back into the meme generator again.  I was able to empty out the text boxes easily with the following:

```
$('.mg-textbox-text').val("");
$('.mg-textbox-text').attr('data-text','');	
```

But emptying out the actual "helpers" is a different story.  I tried using the **deserialize** function, and that sort of worked, except that it throws off the locations of the text boxes I think.  No matter how I use the **deserialize** I never really get the desired effects.  I think the easiest thing would be to just have the helpers start blank, and that would be close enough.

is there a line of code that I could use to clear out the "helpers?".. Bonus would also be to wipe out all layers higher than layer2 .. but I would settle for just clearing helpers for 1&2 ..
